src/resources/Entity/index.ts

```diff
@@ -224,6 +224,42 @@ export interface IEntityGetCreditScoreResponse {
   updated_at: string,
 }
 
+export const CreditScoreStatuses = {
+  completed: 'completed',
+  in_progress: 'in_progress',
+  pending: 'pending',
+  failed: 'failed',
+};
+
+export type IEntityCreditScoreStatuses =
+  | 'completed'
+  | 'in_progress'
+  | 'pending'
+  | 'failed';
+
+export interface IEntityCreditScoresFactorsType {
+  code: string,
+  description: string
```

Comment:
I know this file is pretty inconsistent, but let's make sure to have a comma even on the last item in these object definitions